### ISSUE TITLE
The attached SIP fails to validate during ingest because of XML errors on the descriptive metadata

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#296]
This works fine on the .19 implementation but not on the newest one.
[sip_that_fails_on_validating_ead2002.zip](https://github.com/keeps/roda/files/219267/sip_that_fails_on_validating_ead2002.zip)

**Comment by jmaferreira** [comment]
The error is:
```
ValidationReport [valid=false, message=XML document structures must start and end within the same entity., issues=[]]
```

**Comment by hsilva-keep** [comment]
@jmaferreira can't reproduce and I'm using the last code available.

**Comment by jmaferreira** [comment]
I believe you have reproduce it already.

**Comment by hsilva-keep** [comment]
Basically, as I've already mentioned in issue #283, XML Schemas are loaded every single time a validation is going to be performed. And, when loading them from classpath, between 1) getting the inputstream, 2) loading the schema, 3) closing the schema; and doing this sequence again for the following XML, was generating the error reported with the following exception:
```
2016-04-15 11:27:26,617 [WorkersSystem-akka.actor.default-dispatcher-8] ERROR o.r.c.c.validation.ValidationUtils - Error validating descriptive metadata
org.xml.sax.SAXParseException: XML document structures must start and end within the same entity.
        at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source)
        at org.apache.xerces.util.ErrorHandlerWrapper.fatalError(Unknown Source)
        at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
        at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
        at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source)
        at org.apache.xerces.impl.XMLScanner.reportFatalError(Unknown Source)
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.endEntity(Unknown Source)
        at org.apache.xerces.impl.XMLDocumentScannerImpl.endEntity(Unknown Source)
        at org.apache.xerces.impl.XMLEntityManager.endEntity(Unknown Source)
        at org.apache.xerces.impl.XMLEntityScanner.load(Unknown Source)
        at org.apache.xerces.impl.XMLEntityScanner.scanContent(Unknown Source)
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanContent(Unknown Source)
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source)
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
        at org.apache.xerces.impl.xs.opti.SchemaParsingConfig.parse(Unknown Source)
        at org.apache.xerces.impl.xs.opti.SchemaParsingConfig.parse(Unknown Source)
        at org.apache.xerces.impl.xs.opti.SchemaDOMParser.parse(Unknown Source)
        at org.apache.xerces.impl.xs.traversers.XSDHandler.getSchemaDocument(Unknown Source)
        at org.apache.xerces.impl.xs.traversers.XSDHandler.resolveSchema(Unknown Source)
        at org.apache.xerces.impl.xs.traversers.XSDHandler.constructTrees(Unknown Source)
        at org.apache.xerces.impl.xs.traversers.XSDHandler.parseSchema(Unknown Source)
        at org.apache.xerces.impl.xs.XMLSchemaLoader.loadSchema(Unknown Source)
        at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source)
        at org.apache.xerces.impl.xs.XMLSchemaLoader.loadGrammar(Unknown Source)
        at org.apache.xerces.jaxp.validation.XMLSchemaFactory.newSchema(Unknown Source)
        at javax.xml.validation.SchemaFactory.newSchema(SchemaFactory.java:638)
        at org.roda.core.common.validation.ValidationUtils.validateDescriptiveBinary(ValidationUtils.java:296)
        at org.roda.core.common.validation.ValidationUtils.isAIPMetadataValid(ValidationUtils.java:99)
        at org.roda.core.plugins.plugins.base.DescriptiveMetadataValidationPlugin.execute(DescriptiveMetadataValidationPlugin.java:118)
        at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.executePlugin(DefaultIngestPlugin.java:538)
        at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.executePlugin(DefaultIngestPlugin.java:524)
        at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.doDescriptiveMetadataValidation(DefaultIngestPlugin.java:476)
        at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.execute(DefaultIngestPlugin.java:231)
        at org.roda.core.plugins.orchestrate.akka.AkkaWorkerActor.onReceive(AkkaWorkerActor.java:43)
        at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:165)
        at akka.actor.Actor$class.aroundReceive(Actor.scala:482)
        at akka.actor.UntypedActor.aroundReceive(UntypedActor.scala:95)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:526)
        at akka.actor.ActorCell.invoke(ActorCell.scala:495)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
        at akka.dispatch.Mailbox.run(Mailbox.scala:224)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by hsilva-keep** [comment]
Solved via fb9aed70386734d231d6b2f8d16bce515bd3e5c0
